### ISSUE TITLE
Removing the email lib

### ISSUE THREAD

**Issue by chambas** [RoseSecurity/Rotty.py#4]
Consider using the internal email built into your project.
See here +info:
https://docs.python.org/3/library/email.examples.html

**Comment by RoseSecurity** [comment]
Good call! I am experimenting with more efficient ways to make this work so I will check that out, thank you.
